src/motors.cpp

```diff
@@ -1,28 +1,40 @@
 
 #include "motors.hpp"
+#include "ESC.h"
```

Comment:
what is `ESC.h`?

Reply:
Its the RC_ESC library. I should add a comment

Reply:
Ah, right!